File: test/fizz_buzz_test.rb
require 'fizzbuzz'

class TestFizzBuzz < Test::Unit::TestCase
    def test_3
        assert_equal('fizz', FizzBuzz.result(3))
    end

    def test_5
        assert_equal('buzz', FizzBuzz.result(5))
    end

    def test_13
        assert_equal('fizz', FizzBuzz.result(13))
    end

    def test_15
        assert_equal('fizzbuzz', FizzBuzz.result(15))
    end

    def test_30
        assert_equal('fizzbuzz', FizzBuzz.result(30))
    end
end

File: FizzBuzz.Test/FizzBuzzTest.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace FizzBuzz.Test
{
    [TestClass]
    public class FizzBuzzTest
    {
        [TestMethod]
        public void Test03()
        {
            Assert.AreEqual("fizz", FizzBuzz.GetResult(3));
        }

        [TestMethod]
        public void Test05()
        {
            Assert.AreEqual("buzz", FizzBuzz.GetResult(5));
        }

        [TestMethod]
        public void Test13()
        {
            Assert.AreEqual("fizz", FizzBuzz.GetResult(13));
        }

        [TestMethod]
        public void Test15()
        {
            Assert.AreEqual("fizzbuzz", FizzBuzz.GetResult(15));
        }

        [TestMethod]
        public void Test30()
        {
            Assert.AreEqual("fizzbuzz", FizzBuzz.GetResult(30));
        }

    }
}
